tests/virtual_machines_test.go

```diff
@@ -489,7 +489,7 @@ var _ = Describe("Virtual Machines", func() {
 			})
 		})
 		//2995
```

Comment:
Please link a GitHub Issue where you describe the failure and attach the error log